cypress/support/commands.js

```diff
@@ -40,17 +41,19 @@ Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', pas
   });
 });
 
-Cypress.Commands.add('login', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+Cypress.Commands.add('login', 
+  (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
```

Comment:
The 'login' command is defined twice in this file. Ensure that each custom command has a unique name to avoid conflicts.